### PR TITLE
Add recipe for aadict

### PR DESCRIPTION
**Pull request by nelson-liu**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by nelson-liu** [comment]
@conda-forge/help-python this should be ready for a look

File: recipes/aadict/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - aadict
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://github.com/metagriffin/aadict
+  summary: An auto-attribute dict (and a couple of other useful dict functions)
+  license: GPL-2.0-or-later
```
Comment (dopplershift) [inline recipes/aadict/meta.yaml:37]: Looks like this is actually 3.0 or later

**Review by dopplershift — CHANGES_REQUESTED** [review]
Looks good minus the tweak to the license.

File: recipes/aadict/meta.yaml (hunk 1 above)
Comment (nelson-liu) [inline recipes/aadict/meta.yaml:37]:
```suggestion
  license: GPL-3.0-or-later
```

Comment (nelson-liu) [reply recipes/aadict/meta.yaml]: thanks, fixed!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**